internal/telemetry/telemetry_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package telemetry
```

Comment:
can we add a test for the `telemetry.TrackCommand`?

Reply:
Yes, I've added a test for that in the latest commit now.

Reply:
I see that new test is failing here on Windows... any ideas what might be causing the failure? Or how I can manually try it out on a Windows box?

Reply:
I'm adding some temporary printf's to debug...

Reply:
Filpe, as discussed on Friday, I have split a private "track" function out of the "TrackCommand" function, and I have a unit test for the track function now.

Reply:
> I see that new test is failing here on Windows

For context, what was the windows issue here in the end?